src/frontend/src/views/GroupMembersPage/components/MembersTab/components/ActiveMembersTable/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { Table } from "src/common/components/library/Table";
+import { PersonIconCell } from "../PersonIconCell";
+
+interface Props {
+  members: any;
```

Comment:
why is this any? wouldn't the type be User[] ?

Reply:
this is also a TODO item. I can add a comment